### ISSUE TITLE
Colour contrast readOnly and placeholder text

### ISSUE THREAD

**Issue by dhinrichs-scottlogic** [ScottLogic/prompt-injection#785]
# Feature Request

## Description
Ensure all color contrast passes, including disabled buttons,  readOnly fields (e.g. config inputs) and placeholder text (e.g. chat input)
- adjusted the --main-input-inactive-text-colour to pass but be less close to black
- adjusted the --level-button-colour-disabled colour to provide better contrast (it technically passed before because the text is bold, but just nudging it to also pass 4.5:1).
- Added placeholder pseudo selector to css to adjust textinput placeholder text colour

## Screenshots
![Image](https://github.com/ScottLogic/prompt-injection/assets/125262707/ab0c134a-4603-4363-87ce-f78b42ff5275)
![Image](https://github.com/ScottLogic/prompt-injection/assets/125262707/0fa34269-93f7-42c3-84ec-e297aecb61f9)
![Image](https://github.com/ScottLogic/prompt-injection/assets/125262707/43f2937d-2a58-4fbe-af14-a7b5e93b4e09)

## Additional context
Colour contrast issues for placeholder text are easily missed by automated tools, but colour contrast ratios should still pass for placeholder text and inactive input text.

## Acceptance criteria

**GIVEN** text/number inputs and buttons
**WHEN** they are disabled, readonly or have placeholder text
**THEN** the text to background contrast ratio is at least 4.5:1
